jaws-journey-deploy/orb.yml

```diff
@@ -1008,6 +1008,9 @@ jobs:
     working_directory: ~/working
     executor: python
     parameters:
+      manifest_repository:
+        type: string
+        default: ${CIRCLE_PROJECT_USERNAME}/${CIRCLE_PROJECT_REPONAME}
```

Comment:
Do we want to open the possibility of pulling manifests from a different GitHub project username? Could keep $(CIRCLE_PROJECT_USERNAME) in the update_helm.sh script, and just allow the user to specify the repo name